chdkptp/lua.py

```diff
@@ -25,7 +24,7 @@ class LuaContext(object):
         inside of `pcall` and raises proper Exceptions.
     """
     def _raise_exception(self, errval):
-        if isinstance(errval, (basestring, numbers.Number)):
+        if isinstance(errval, (str, numbers.Number)):
```

Comment:
Have you checked if this works with Python 2? I'm not sure if lupa maps Lua strings to the `str` or `unicode` type. If it's the latter, we might need a workaround to maintain support for Python 2.x.

Reply:
Yeah, I'm not sure about that, I read somewhere just to replace the basestring to str, I'm going to do a deeper research about it, just to be sure.